### PR TITLE
SAMSON-393 bump template limit so we can store endless resources

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/bre @zendesk/compute

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -584,8 +584,8 @@
     t.text "ca_cert"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["name"], name: "index_vault_servers_on_name", unique: true, length: 191
     t.boolean "versioned_kv", default: false, null: false
+    t.index ["name"], name: "index_vault_servers_on_name", unique: true, length: 191
```
Comment (grosser) [inline db/schema.rb]: fallout from earlier merge conflict, indexes always go last

**Review by ragurney — APPROVED** [review]

File: plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class BumpLimit < ActiveRecord::Migration[5.2]
+  def change
+    change_column :kubernetes_release_docs, :resource_template, :text, limit: 1073741823
```
Comment (adammw) [inline plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: what is this magic number, and why doesn't it match the number in the schema.rb?

Comment (grosser) [reply plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: max mysql column length

File: db/schema.rb [hunk 3]
```diff
@@ -235,7 +235,7 @@
     t.integer "deploy_group_id"
     t.decimal "limits_cpu", precision: 6, scale: 2, null: false
     t.integer "limits_memory", null: false
-    t.text "resource_template"
+    t.text "resource_template", limit: 4294967295
```
Comment (grosser) [inline db/schema.rb]: that's a bug and will be caught by cleanliness test :)

File: plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb (hunk 2 above)
Comment (adammw) [reply plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: comment / use a constant?

Comment (yizhang-zen) [reply plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: Same question on the numbers, is it safe to set "unlimited" limits?

Comment (grosser) [reply plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: 🤷‍♂️ not super safe, but we have been doing it for job output all these years and nothing bad happened

Comment (grosser) [reply plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb]: comment added

**Review by craig-day — APPROVED** [review]

**Pull request merged**